src/tests/ftest/pool/evict_test.py

```diff
@@ -38,110 +39,257 @@ class EvictTest(TestWithServers):
 
     def test_evict(self):
         """
-        Test evicting a client to a pool.
+        Test evicting a client from a pool.
 
         :avocado: tags=pool,poolevict,quick
         """
-        size = self.params.get("size", '/run/tests/sizes/size1gb/*')
-        setid = self.params.get("setname", '/run/tests/setnames/validsetname/*')
-        connectperm = self.params.get("perms",
-                                      '/run/tests/connectperms/permro/*')
+        pool = []
+        container = []
+        cont_uuid = []
+        non_pool_servers = []
+        # Target list is configured so that the pools are across all servers
+        # except the pool under test is created on half of the servers
+        pool_tgt = [num for num in range(len(self.hostlist_servers))]
+        pool_tgt_ut = [num for num in range(int(len(self.hostlist_servers)/2))]
+        tlist = [pool_tgt, pool_tgt, pool_tgt_ut]
+        pool_servers = [self.hostlist_servers[:len(tgt)] for tgt in tlist]
+        # Create TestPool
+        for count, _ in enumerate(tlist):
+            self.log.info("Create a pool")
+            pool.append(TestPool(self.context, self.log))
+            pool[count].get_params(self)
+            pool[count].name.value = self.server_group
+            pool[count].target_list.value = tlist[count]
+            self.log.info(
+                "Pool # %s target list is %s",
+                count+1, pool[count].target_list)
+            pool[count].create()
+            self.log.info("Pool # %s UUID is %s and handle is %s",
```

Comment:
FYI, The latest (in PR #701) will display this information when you call TestPool.create().